codecov.yml

```diff
@@ -1,4 +1,6 @@
 ---
+ignore:
+  - "sql/docker_interface.go"
```

Comment:
🤔 Why does codecov complain about this? Aren't we using this in our tests and code?

Reply:
No, we don't use them since we're mocking those method implementations. So, they don't get called and those lines miss out of coverage.

Reply:
Alright, thanks 👍